native/CMakeLists.txt

```diff
@@ -2020,7 +2179,7 @@ if(USE_EDIT_BOX)
     )
 endif()
 
-if(ANDROID OR IOS OR OHOS)
+if(NOT OPENHARMONY AND (ANDROID OR IOS OR OHOS))
```

Comment:
Can `OPENHARMONY` and `OHOS` be set at the same time ?

Reply:
Yes, ohos will be defined internally, so in some places it is necessary to judge Openharmony first and then judge OHOS

Reply:
ok